### PR TITLE
Finish-Desktop-Version

### PR DESCRIPTION
**Pull request by muneeb3123**
THIS PR INCLUDES:

**MEDIA QUERIES** for responsiveness.
**MENU BAR**  FOR DESKTOP VERSION
**SWAP CARD** ORDER FOR EVERY PAIR OF CARD
**LINTERS** FIXED ALL ERRORS

THANKS FOR REVIEWING

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,67 +146,68 @@ <h2 class="content-title">Multi-Post Stories</h2>
             </div>
```
Comment (pinkmoon25) [inline index.html:146]: - [ ] Great work implementing the desktop version, you have done really well, As it is required for us to stick to the original figma design as much as possible I have some suggestions where we can improve, such as fixing the ` font-size ` and `line-height` property for your project cards title, based on the figma it should be 40px and 52px respectively :point_down: 
image

image

File: style.css [hunk 2]
```diff
@@ -431,6 +375,13 @@ nav #mylogo {
   top: 1%;
 }
 
+.drop-box {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  right: 4%;
+}
+
```
Comment (pinkmoon25) [inline style.css:384]: - [ ] Please notice that the technologies or languages cards in the About me sections are placed vertically, they are supposed to be aligned horizontally please see :point_down: 
image

how they should be :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -529,3 +480,250 @@ nav #mylogo {
...
+  .contact-form {
+    margin: 50px auto;
+    width: 50%;
+  }
+
+  .contact-form > input,
+  .contact-form > textarea {
+    width: 100%;
+  }
```
Comment (pinkmoon25) [inline style.css:574]: - [ ] Please notice the get in touch button is centered in the original design, yours is aligned to the left please see :point_down: 
image

how it should be :point_down: 
image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @muneeb3123 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark:
github flow :heavy_check_mark: 
follows figma template :heavy_check_mark: 
media queries :heavy_check_mark: 
responsive :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @muneeb3123 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️ 
 - Responsive design. ✔️ 
 - Correct use of GitHub flow. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**